src/eip4824.ts

```diff
@@ -0,0 +1,155 @@
+import express from 'express';
+import { getSpace } from './helpers/spaces';
+import db, { sequencerDB } from './helpers/mysql';
+
+const router = express.Router();
+
+router.get('/:space', async (req, res) => {
+  let space: any = {};
+  const baseUrl = `${req.protocol}://${req.get('host')}${req.originalUrl}`;
```

Comment:
<img width="728" alt="Untitled 12" src="https://github.com/snapshot-labs/snapshot-hub/assets/15967809/cd3ba417-5a57-4c23-a9c0-064ba882784f">
Is it because of the port forward? URLs still have localhost

Reply:
You mean the "https://"? To me it give "http" locally

Reply:
no i mean for host `localhost` vs `devtonnels.ms` in URL

Reply:
Can you try again?